end-to-end-tests-localdb/runtime-config/get_pullrequest_info.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
```

Comment:
@pvannierop i like this. I guess we're doing something like this in the current e2e tests. Maybe also here there's a way to share this code

Reply:
See post above. I could place this script in the `common` dir.

Reply:
File was moved to `shared` directory.